tests/contract/test_research_paper_parser_chunker.py

```
@@ -68,3 +73,62 @@ def test_markdown_paper_parser_rejects_missing_file(tmp_path):
+    missing = tmp_path / "missing.md"
+    result = _run_sample_script(
+        ["--paper", str(missing)],
+        cwd=REPO_ROOT,
+    )
+
+    assert result.returncode == 1
+    assert "EmptyCorpus" in result.stderr
+
+
+def test_parse_paper_sample_script_works_outside_repo_root(tmp_path):
+    """! @brief 默认论文路径必须不依赖运行目录，从仓库外也能跑通。"""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)